src/Core/Domain/OrderReturn/QueryHandler/GetOrderReturnForEditingHandlerInterface.php

```diff
@@ -0,0 +1,43 @@
+<?php
+/**
+ * 2007-2020 PrestaShop SA and Contributors
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2020 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ */
+
+namespace PrestaShop\PrestaShop\Core\Domain\OrderReturn\QueryHandler;
+
+use PrestaShop\PrestaShop\Core\Domain\OrderReturn\Query\GetOrderReturnForEditing;
+use PrestaShop\PrestaShop\Core\Domain\OrderReturn\QueryResult\EditableOrderReturn;
+
+/**
+ * Defines contract for GetOrderReturnForEditingHandler
+ */
+interface GetOrderReturnForEditingHandlerInterface
+{
+    /**
+     * @param GetOrderReturnForEditing $query
+     *
+     * @return EditableOrderReturn
+     */
+    public function handle(GetOrderReturnForEditing $query): EditableOrderReturn;
```

Comment:
Just for consistency- i thin we call most of these 'somethimgForEditing', not EditableSomething, but not sure. And its not in docs I guess so i doubt its required 🤷‍♂️

Reply:
I am pretty sure I just copied idea from one of other classes. But if we are moving to use  `ForEditing` instead of `Editable` that's fine for me. I guess `ForEditing` at least uses the same naming as handler.